src/scoreSheetBot.py

```diff
@@ -47,7 +47,7 @@ def _set_current(self, ctx: Context, battle: Battle):
         self.battle_map[key_string(ctx)] = battle
 
     def _clear_current(self, ctx):
-        self.battle_map[key_string(ctx)] = None
+        self.battle_map.pop(key_string(ctx), None)
```

Comment:
This isn't the same, but I assume we don't need to keep a reference to cleared keys